Storage/SQL/SQLiteBookmarksSyncing.swift

```diff
@@ -834,31 +853,34 @@ extension SQLiteBookmarks {
     fileprivate func structureQueryForTable(_ table: String, structure: String) -> String {
         // We use a subquery so we get back rows for overridden folders, even when their
         // children aren't in the shadowing table.
-        let sql =
-        "SELECT s.parent AS parent, s.child AS child, COALESCE(m.type, -1) AS type " +
-        "FROM \(structure) s LEFT JOIN \(table) m ON s.child = m.guid AND m.is_deleted IS NOT 1 " +
-        "ORDER BY s.parent, s.idx ASC"
+        let sql = """
+            SELECT s.parent AS parent, s.child AS child, COALESCE(m.type, -1) AS type
+            FROM \(structure) s LEFT JOIN \(table) m ON
+                s.child = m.guid AND
+                m.is_deleted IS NOT 1
+            ORDER BY s.parent, s.idx ASC
+            """
+
         return sql
     }
 
     fileprivate func remainderQueryForTable(_ table: String, structure: String) -> String {
-        // This gives us value rows that aren't children of a folder.
-        // You might notice that these complementary LEFT JOINs are how you
-        // express a FULL OUTER JOIN in sqlite.
-        // We exclude folders here because if they have children, they'll appear
-        // in the structure query, and if they don't, they'll appear in the bottom
-        // half of this query.
-        let sql =
-        "SELECT m.guid AS guid, m.type AS type " +
-        "FROM \(table) m LEFT JOIN \(structure) s ON s.child = m.guid " +
-        "WHERE m.is_deleted IS NOT 1 AND m.type IS NOT \(BookmarkNodeType.folder.rawValue) AND s.child IS NULL " +
-
-        "UNION ALL " +
-
-        // This gives us folders with no children.
-        "SELECT m.guid AS guid, m.type AS type " +
-        "FROM \(table) m LEFT JOIN \(structure) s ON s.parent = m.guid " +
-        "WHERE m.is_deleted IS NOT 1 AND m.type IS \(BookmarkNodeType.folder.rawValue) AND s.parent IS NULL "
+        let sql = """
+            -- This gives us value rows that aren't children of a folder.
+            -- You might notice that these complementary LEFT JOINs are how you
+            -- express a FULL OUTER JOIN in sqlite.
+            -- We exclude folders here because if they have children, they'll appear
+            -- in the structure query, and if they don't, they'll appear in the bottom
+            -- half of this query.
```

Comment:
Leave this big comment as a Swift comment — no point making SQLite parse it every time.